cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -359,6 +359,16 @@ objects:
             version_constraints:
               min_version: 4.11.0
 
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: azure-runtime-config
+  data:
+    config.json: |
+      {
+        "cloudEnvironment": "AzurePublicCloud"
+      }
```

Comment:
In CS, we have a parameter that takes a json obj and set it here (see [MR#8478](https://gitlab.cee.redhat.com/service/uhc-clusters-service/-/merge_requests/8478/diffs#24149fd14e0b7c80e0bcb139f89e767ed43c806f_332_340))
```
config.json: ${AZURE_RUNTIME_CONFIG}
```

I see that the rest of the configmap values are hardcoded here so i've kept it the same. Is there a preference to parameterise this? Additional configuration that would be added here for example would be ACR configuration that's used in CS (see related jira issue [ARO-9331](https://issues.redhat.com/browse/ARO-9331)).

Reply:
once additional dynamic configuration is required for an env, you can do something like this here 

https://github.com/Azure/ARO-HCP/blob/129323a66958d8122ecacecd1350af3e493967f2/maestro/deploy/helm/agent/templates/maestro.secret.yaml

so basically templating individual inner fields of the json value.

Reply:
ah perfect! I think we can do this as a follow on task, for now we can start with the hardcoded configmap 👍 I'll create a jira issue to track this.